packages/fs-detectors/src/workspaces/get-workspace-package-paths.ts

```diff
@@ -33,6 +33,12 @@ export async function getWorkspacePackagePaths({
     case 'pnpm':
       results = await getPnpmWorkspacePackagePaths({ fs: workspaceFs });
       break;
+    case 'rush':
+      // /common/temp is the location that Rush defaults adding the pnpm workspace file
+      results = await getPnpmWorkspacePackagePaths({
+        fs: fs.chdir('/common/temp'),
```

Comment:
Seeing how this can be used, it my be more robust to use projects enumerated in `rush.json`

https://github.com/microsoft/rushstack/blob/8671d79a48b2a87bcf139567ab8281e50d00c9b7/libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts#L126-L137